MJ68-sleep.ict

```diff
@@ -499,52 +546,66 @@ rooms:
               });
           }
       })
-    onstep: |
-      // On tap action, animate the arm so that the hand slaps the tapped area
-      if (ct.actions.Tap.pressed) {
-          // Get direction of tap
-          let direction = ct.u.pointDirection(448, 320, ct.touch.x, ct.touch.y);
-          // Rotate arm towards tap
-          this.arm.rotation = direction - 180;
-          
-          // Get distance of tap
-          let distance = ct.u.pointDistance(448, 320, ct.touch.x, ct.touch.y);
-          // Scale so hitbox ends on tap
-          if (distance > 334) {
-              this.arm.scale.x = 1;
-              this.arm.scale.y = 1;
+    onstep: |2-
+          // On tap action, animate the arm so that the hand slaps the tapped area
+          if (ct.actions.Tap.pressed) {
+              // Get direction of tap
+              let direction = ct.u.pointDirection(448, 320, ct.touch.x, ct.touch.y);
+              // Rotate arm towards tap
+              this.arm.rotation = direction - 180;
+              
+              // Get distance of tap
+              let distance = ct.u.pointDistance(448, 320, ct.touch.x, ct.touch.y);
+              // Scale so hitbox ends on tap
+              if (distance > 334) {
+                  this.arm.scale.x = 1;
+                  this.arm.scale.y = 1;
+              }
+              else {
+                  let shortScale = distance/334;
+                  this.arm.scale.x = shortScale;
+                  this.arm.scale.y = shortScale;
+              }
+
+              // Determine if mosquito was under the hand
+              if (ct.place.occupied(this.arm, 'Bug')) {
+                  this.mosquito.kill = true;
+              }
           }
-          else {
-              let shortScale = distance/334;
-              this.arm.scale.x = shortScale;
-              this.arm.scale.y = shortScale;
+
+          // On release, put hand back under cover
+          if (ct.actions.Tap.released) {
+              this.arm.rotation = 170;
+              this.arm.scale.x = 0.5;
+              this.arm.scale.y = 1;
           }
 
-          // Determine if mosquito was under the hand
-          if (ct.place.occupied(this.arm, 'Bug')) {
-              this.mosquito.kill = true;
+          // Change bgs based off game statuses
+          if (!this.mosquito.kill) {
+              ct.backgrounds.add('furrowed', 0, -9);
           }
-      }
 
-      // On release, put hand back under cover
-      if (ct.actions.Tap.released) {
-          this.arm.rotation = 170;
-          this.arm.scale.x = 0.5;
-          this.arm.scale.y = 1;
-      }
+          if (this.gameTimer.timeLeft == 0 || this.mosquito.kill) {
+              if (this.won) {
+                  ct.backgrounds.add('happy', 0, -7); 
+              }
+              else {
+                  ct.backgrounds.add('lose', 0, -8);
+              }        
+          }
```

Comment:
You're already fixing this, thanks

Reply:
Fixed in [d038378](https://github.com/annedo/Minijam68-sleep/pull/2/commits/d038378399a1b607341f7a75908fdaab5217cb83)